practice_app/apps/article/views.py

```diff
@@ -0,0 +1,179 @@
+from asyncio.windows_events import NULL
```

Comment:
This line causes an error on Linux. As we will use docker to publish the application and NULL is not used in this file, please remove this line.

Reply:
Looks like I forgot to delete it. Now it is removed.